src/Template/Layout/js/app/directives/datepicker.js

```diff
@@ -58,6 +58,8 @@ export default {
 
                 try {
                     let datePicker = flatpickr(element, options);
+                    element.dataset.originalValue = element.value;
+                    // element._flatpickr = datePicker;
```

Comment:
remove commented line?

Reply:
that is part of a refactor (did it here cause i needed it in this branch)